packages/x-components/src/x-modules/next-queries/store/__tests__/actions.spec.ts

```diff
@@ -35,7 +36,7 @@ describe('testing next queries module actions', () => {
     NextQueriesGetters,
     NextQueriesMutations,
     NextQueriesActions
-  > = new Store(nextQueriesXStoreModule as any);
+  > = new Store(nextQueriesXStoreModule as XStoreModule<any, any, any, any>);
```

Comment:
You can undo those changes, they are not needed.

Reply:
Sure? ESLint warned `Unsafe argument of type 'any' assigned to a parameter`